packaging/windows/msbuildextensions/variables.wxi

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<Include xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <?define Servicing_Key_SP   =   "0" ?>
+  <?define Servicing_Key_SPIndex   =   "0" ?>
+  <?define Servicing_Key_SPName   =   "Beta" ?>
+  <?define Dotnet_ProductVersion   =   "$(var.BuildVersion)" ?>
+  <?define Dotnet_DisplayVersion   =   "$(var.DisplayVersion)" ?>
+  <?define Dotnet_BuildVersion   =   "$(var.BuildVersion)" ?>
+  <?define Manufacturer     =   "Microsoft Corporation" ?>
+  <?define ProductName      =   "$(var.ProductMoniker) ($(sys.BUILDARCH))" ?>
+  <?define ProductLanguage  =   "1033" ?>
+  <?define ProductVersion   =   "$(var.Dotnet_ProductVersion)" ?>
+  <?define ProductFamily    =   "dotnet" ?>
+  <?define ProductEdition   =   "001dotnet" ?>
+  <?define LCID  = "$(var.ProductLanguage)"?>
+  <?define DowngradeErrorMessage  = "A newer version is already installed; please uninstall it and re-run setup."?>
+
+  <?define Platform   =   "$(sys.BUILDARCH)" ?>
+  <?if $(var.Platform)=x86?>
+  <?define Program_Files="ProgramFilesFolder"?>
+  <?define Win64AttributeValue=no?>
+  <?elseif $(var.Platform)=x64?>
+  <?define Program_Files="ProgramFiles64Folder"?>
```

Comment:
Fixed this already. Only doing it for x86. In fact, this version already does that, because we condition generating the MSI on x86. The code above was a copy/paste that I didn't remove :D. Anyways, removed now.